frappe/modules.txt

```diff
@@ -8,5 +8,4 @@ Desk
 Integrations
 Printing
 Contacts
-Social
```

Comment:
Because this is removed the doctypes are not getting removed automatically. We have to handle this in `remove_orpahn_doctypes` 

![Uploading image.png…]()

Reply:
(resolved in my last commit)